src/fetchMissing.js

```diff
@@ -1,12 +1,17 @@
-const fetch = require('node-fetch')
-const utils = require('web3-utils')
-
+const fetch = require('node-fetch');
+const utils = require('web3-utils');
+const fs = require('fs');
 async function getLogo(xdaiAddr) {
   const response = await fetch(`https://blockscout.com/poa/xdai/tokens/${xdaiAddr}/token-transfers`)
   const text = await response.text()
-  const mainnetAddr = text.match(/https:\/\/etherscan.io\/token\/(0x[a-fA-F0-9]{40})/)[1]
-  const formattedAddr = utils.toChecksumAddress(mainnetAddr)
-  return `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${formattedAddr}/logo.png`
+  const mainnetAddr = text.match(/https:\/\/etherscan.io\/token\/(0x[a-fA-F0-9]{40})/)
+  if (mainnetAddr) {
+    const formattedAddr = utils.toChecksumAddress(mainnetAddr[1])
+    return formattedAddr
```

Comment:
```suggestion
    return utils.toChecksumAddress(mainnetAddr[1])
```

Let's return the function call directly